consensus/abci.go

```diff
@@ -94,127 +89,110 @@ func (abci *TendermintABCI) Info(req tmtAbciTypes.RequestInfo) tmtAbciTypes.Resp
 	if err != nil {
-		// nolint: errcheck
 		abci.logger.Error("Error getting latest database state", "err", err)
+		return tmtAbciTypes.ResponseCommit{}
 	}
 
 	ethState, err := abci.getCurrentDBState()
 	if err != nil {
-		abci.logger.Error("Error getting latest state", "err", err) // nolint: errcheck
+		abci.logger.Error("Error getting latest state", "err", err)
+		return tmtAbciTypes.ResponseCommit{}
 	}
 
-	abci.logger.Info("TendermintABCI::Commit()", "blockHash", blockHash.Hex())
+	abci.logger.Info("Committing state", "blockHash", blockHash.Hex())
+
 	rootHash := abci.getCurrentBlock().Root()
 	abci.checkTxState = ethState.Copy()
 	if err != nil {
-		abci.logger.Error("Error committing latest state", "err", err) // nolint: errcheck
+		abci.logger.Error("Error committing latest state", "err", err)
+		return tmtAbciTypes.ResponseCommit{}
 	}
-	// The app should respond to the Commit request with a byte array, which is the deterministic state root of the
-	// application. It is included in the header of the next block. It can be used to provide easily verified
-	// Merkle-proofs of the state of the application.
+
 	return tmtAbciTypes.ResponseCommit{Data: rootHash.Bytes()}
 }
 
 // Query queries the state of the TendermintABCI
 func (abci *TendermintABCI) Query(query tmtAbciTypes.RequestQuery) tmtAbciTypes.ResponseQuery {
-	abci.logger.Info("TendermintABCI::Query()", "data", query)
-	abci.logger.Debug("Query") // nolint: errcheck
+	abci.logger.Info("Querying state", "data", query)
 	var in jsonRequest
 	if err := json.Unmarshal(query.Data, &in); err != nil {
-		return tmtAbciTypes.ResponseQuery{Code: uint32(abciTypes.ErrEncodingError.Code),
-			Log: err.Error()}
+		return tmtAbciTypes.ResponseQuery{Code: uint32(abciTypes.ErrEncodingError.Code), Log: err.Error()}
 	}
 	var result interface{}
 	if err := abci.ethRPCClient.Call(&result, in.Method, in.Params...); err != nil {
-		return tmtAbciTypes.ResponseQuery{Code: uint32(abciTypes.ErrInternalError.Code),
-			Log: err.Error()}
+		return tmtAbciTypes.ResponseQuery{Code: uint32(abciTypes.ErrInternalError.Code), Log: err.Error()}
 	}
 	resultBytes, err := json.Marshal(result)
 	if err != nil {
-		return tmtAbciTypes.ResponseQuery{Code: uint32(abciTypes.ErrInternalError.Code),
-			Log: err.Error()}
+		return tmtAbciTypes.ResponseQuery{Code: uint32(abciTypes.ErrInternalError.Code), Log: err.Error()}
 	}
+
 	return tmtAbciTypes.ResponseQuery{Code: tmtAbciTypes.CodeTypeOK, Value: resultBytes}
 }
 
-//-------------------------------------------------------
-
 // validateTx checks the validity of a tx against the blockchain's current state.
 // it duplicates the logic in database's tx_pool
 func (abci *TendermintABCI) validateTx(tx *ethTypes.Transaction) tmtAbciTypes.ResponseCheckTx {
-	abci.logger.Info("TendermintABCI::validateTx()", "data", tx.Hash().String())
-	// Heuristic limit, reject transactions over 32KB to prevent DOS attacks
+	abci.logger.Info("Validating TX", "data", tx.Hash().String())
```

Comment:
Why this log is kept? Suggested to use `Debug()` instead

Reply:
Because it's actually printing something useful, the txHash